### ISSUE TITLE
Extension messes with file saving, error after Uninstall

### ISSUE THREAD

**Issue by qubodup** [berwynhoyt/wordcount-sections#2]
After uninstalling the extension, I get an error:
```
A Scripting Framework error occurred while running the Basic script vnd.sun.star.script:Wordcounter.wordcount_sections.on_save?language=Basic&location=application.

Message: The following Basic script could not be found:
library: 'Wordcounter'
module: 'wordcount_sections'
method: 'on_save'
location: 'application'
```
![error](https://github.com/berwynhoyt/wordcount-sections/assets/18312/56ec84ba-58af-4af0-8c1a-462b80d983a5)

Steps to reproduce:
1. Go to %APPDATA%\libreoffice\
2. Rename the folder "4" to "4x"
3. Open writer
4. Install the extension
5. Restart writer
6. Open a writer document, save it - no issue (but also the bar at the bottom shows nothing because of the save progress bar)
7. Uninstall the extension
8. Restart writer
9. Open a writer document, save it - Error pops up

Version: 7.4.4.2 (x64) / LibreOffice Community
Build ID: 85569322deea74ec9134968a29af2df5663baa21
CPU threads: 16; OS: Windows 10.0 Build 19045; UI render: Skia/Raster; VCL: win
Locale: en-US (en_DE); UI: en-US
Calc: CL

**Comment by berwynhoyt** [comment]
Interesting. Thanks for bringing this to my attention. I could fix by writing some cleanup code except that I don't know how to get LibreOffice to call my cleanup code when someone uninstalls the extension. I've [asked this question on the OO forum](https://forum.openoffice.org/en/forum/viewtopic.php?t=109952).

In the meantime, you can remove the offending event setting manually by opening the LibreOffice base program (not writer), and then going to Tools->Customize. Select the Events tab and you should be able to see an Event assigned to SaveDocument part way down. Click it and then click the Remove button and the OK button.

I'll post an update here if someone helpful replies to my forum post.

**Comment by berwynhoyt** [comment]
This has now been fixed in version 1.2
Thanks to [JeJe for the suggested code](https://forum.openoffice.org/en/forum/viewtopic.php?p=537864#p537864) that led to this fix.
